src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowSnykScanListenerLS.kt

```diff
@@ -155,45 +162,102 @@ class SnykToolWindowSnykScanListenerLS(
-        issues: Map<SnykFile, List<ScanIssue>>
+        issues: List<ScanIssue>,
+        securityIssuesCount: Int? = null,
+        fixableIssuesCount: Int? = null,
     ) {
-        fun navigateToSource(virtualFile: VirtualFile, textRange: TextRange): () -> Unit = {
-            io.snyk.plugin.navigateToSource(project, virtualFile, textRange.startOffset, textRange.endOffset)
+        // only add these info tree nodes to Snyk Code security vulnerabilities for now
+        if (securityIssuesCount == null) {
+            return
         }
+
+        val settings = pluginSettings()
+        // only add these when we enable the consistent ignores flow for now
+        if (!settings.isGlobalIgnoresFeatureEnabled) {
+            return
+        }
+
+        var text = "✅ Congrats! No vulnerabilities found!"
+        val issuesCount = issues.size
+        val ignoredIssuesCount = issues.count { it.isIgnored() }
+        if (issuesCount != 0) {
+            if (issuesCount == 1) {
+                text = "$issuesCount vulnerability found by Snyk"
+            } else {
+                text = "✋ $issuesCount vulnerabilities found by Snyk"
+            }
+            text += ", $ignoredIssuesCount ignored"
+        }
+        rootNode.add(
+            InfoTreeNode(
+                text,
+                project,
+            ),
+        )
+
+        if (fixableIssuesCount != null) {
+            if (fixableIssuesCount > 0) {
+                rootNode.add(
+                    InfoTreeNode(
+                        "⚡\uFE0F $fixableIssuesCount vulnerabilities can be fixed by Snyk DeepCode AI",
+                        project,
+                    ),
+                )
+            } else {
+                rootNode.add(
+                    InfoTreeNode("There are no vulnerabilities fixable by Snyk DeepCode AI", project),
+                )
+            }
+        }
+    }
+
+    private fun displayResultsForRootTreeNode(
+        rootNode: DefaultMutableTreeNode,
+        issues: Map<SnykFile, List<ScanIssue>>,
+    ) {
+        fun navigateToSource(
+            virtualFile: VirtualFile,
+            textRange: TextRange,
+        ): () -> Unit =
+            {
+                io.snyk.plugin.navigateToSource(project, virtualFile, textRange.startOffset, textRange.endOffset)
+            }
         issues
             .filter { it.value.isNotEmpty() }
             .forEach { entry ->
-                val productType = when (rootNode) {
-                    is RootQualityIssuesTreeNode -> ProductType.CODE_QUALITY
-                    is RootSecurityIssuesTreeNode -> ProductType.CODE_SECURITY
-                    is RootOssTreeNode -> ProductType.OSS
-                    is RootContainerIssuesTreeNode -> ProductType.CONTAINER
-                    is RootIacIssuesTreeNode -> ProductType.IAC
-                    else -> throw IllegalArgumentException(rootNode.javaClass.simpleName)
-                }
+                val productType =
```

Comment:
I installed `ktlint` in IntelliJ since I kept having problems with linting, so this file got automatically linted and formatted

Reply:
Ah, question answered. Actually, our editorconfig should take care of it 🤷

Reply:
I'll remove it! Thanks Basti